landscape/how-to/install-landscape-server-on-microsoft-azure.md

```diff
@@ -0,0 +1,205 @@
+# How to install Landscape Server on Microsoft Azure
+
+This guide provides an example of how to install and set up your Landscape server on Microsoft Azure with [cloud-init](https://cloudinit.readthedocs.io/en/latest/). The instructions here can be used for both FIPS-hardened or non-hardened systems.
```

Comment:
```suggestion
This guide provides an example of how to install and set up your Landscape server on Microsoft Azure with [cloud-init](https://cloudinit.readthedocs.io/en/latest/).
```
```suggestion
This guide provides an example of how to install and set up your Landscape server on Microsoft Azure with [cloud-init](https://cloudinit.readthedocs.io/en/latest/). The instructions here can be used for standard or FIPS-compliant deployments.
```

Reply:
(You can ignore the first suggestion of deleting the last sentence.. not sure why it's there... :sweat_smile:)